go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/dnaeon/go-vcr v1.0.1
+	github.com/google/go-querystring v1.0.0
```

Comment:
I'm more cognizant of this module import now that we've removed all other imports (if you exclude test-only imports).

Reply:
We are only using this for `query.Values`.  Perhaps we can copy that function or find a standard-lib alternative?

Reply:
`Values()` is the only exported function from google/go-querystring, and it's using all the other code from the lib, so copying code out is not really an option:
https://github.com/google/go-querystring/blob/master/query/encode.go#L113

I will take a look what would it take to do this with url.Values or something standard.

Reply:
https://golang.org/pkg/net/url/#Values

Reply:
@displague I removed the google/go-querystring dep. I will run some tests still, but you can check the code.

Reply:
Tests succeed.